Podfile

```diff
@@ -1,5 +1,15 @@
 use_frameworks!
 
+target 'OAuthSwift' do
+    platform :ios, '8.3'
+    pod 'SwiftyRSA', '~> 1.2.0'
+end
+
```

Comment:
By becoming dependant and make OAuthSwift target dependant on this I suspect that the project file has been updated to work only with cocoapod
So opening the projet without the workspace will failed
And then Carthage could not work.

I could do the fix later

Then for macOS? :p